### ISSUE TITLE
Tightening indentation rules

### ISSUE THREAD

**Issue by nareshbhatia** [palantir/tslint#3039]
[This is not a bug, it's a question. Please let me know if I should be asking questions elsewhere.]

I am finding that tslint does not complain about bad indentaion. For example, the following code passes with no warnings:

```
class App extends React.Component<{}, {}> {
    private render() {
return (
  "Hello World"
);
}
}

export default App;
```

Here's my tslint.json

```
{
    "extends": "tslint:recommended",
    "rules": {
        "indent": [ true, "spaces", 4 ]
    }
}
```
The `indent` rule simply seems to enforce spaces vs. tabs. It is not even complaining about the 2 space indent of "Hello World".

How can I tighten indentation rules?

**Comment by denyo** [comment]
Could be a duplicate of #2814

**Comment by nareshbhatia** [comment]
@denyo, you are correct. I am closing this one.

As an aside, I am now relying on [prettier](https://github.com/prettier/prettier) to make sure indentation is consistent across the team automatically. You can see my integration at https://github.com/archfirst/react-mobx-mui-ts-seed.
